src/pass.cpp

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2024 <mbernard>************************************************* */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   pass.cpp                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mbernard <mbernard@student.42lyon.fr>      +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/10/24 09:46:04 by mbernard          #+#    #+#             */
+/*   Updated: 2024/10/29 14:27:49 by mbernard         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../includes/Parser.hpp"
+
+bool Parser::verifyPassword(std::string arg, std::string psd, Client& client) {
```

Comment:
Check if the client is already registered so that the Server sends `462 ERR_ALREADYREGISTERED` in case of PASS command attempts from already registered clients.